test/Numerics/DGmethods/compressible_Navier_Stokes/density_current_model.jl

```diff
@@ -130,10 +130,7 @@ function run(
     model = AtmosModel{FT}(
         AtmosLESConfigType,
         param_set;
-        ref_state = HydrostaticState(
-            DryAdiabaticProfile(typemin(FT), FT(300)),
-            FT(0),
-        ),
+        ref_state = HydrostaticState(DryAdiabaticProfile(FT(300)), FT(0)),
```

Comment:
Looks like `DryAdiabaticProfile` only has one argument now, so `FT(0)` I'd think can be omitted as a second argument.

Reply:
`DryAdiabaticProfile` does have one argument, what looks like the second argument is actually the second argument for `HydrostaticState`.